### PR TITLE
ci: enable pyinstaller builds upon gh release tagged

### LINKED ISSUE #178: use gh actions to build nightly binaries
like we did with flare-floss

### PR DESCRIPTION
**Pull request by williballenthin**
unfortunately, haven't tested the "upload to release page" functionality yet, since we don't have a tag.

closes #178

### CODE REVIEW

**Comment by williballenthin** [comment]
example successful run: https://github.com/fireeye/capa/actions/runs/195125848

**Comment by williballenthin** [comment]
"artifacts": image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+jobs:
+  build:
+    name: PyInstaller for ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-16.04
+            # use old linux so that the shared library versioning is more portable
```
Comment (Ana06) [inline .github/workflows/build.yml:16]: 👍

**Comment by Ana06** [comment]
are we releasing a new version? 😄

**Comment by williballenthin** [comment]
looks good when triggered from gh release

image

**Comment by mr-tz** [comment]
Awesome!

**Pull request merged**